doc/api_assets/style.css

```
@@ -67,16 +114,16 @@ a.type {
 a:link,
 a:active,
 a:visited {
-  color: #43853d;
+  color: var(--green1);
```

nit: we could use a CSS variable to simplify this
```suggestion
  color: var(--color-links);
```